arch/arm/src/armv7-a/arm_vectors.S

```diff
@@ -147,7 +157,11 @@ arm_vectorirq:
 	 * and r14.
 	 */
 
-	ldr		r13, .Lirqtmp
+	cmp     r13, #0                 /* IRQ stack is set? */
```

Comment:
Does cpu init r13 to zero after power up?

Reply:
@xiaoxiang781216 

Yes. As long as I confirmed with sabre-6quad dev board and qemu-5.2.0, the initial interrupt stack register for all CPUs was zero.